mlrun/api/crud/model_endpoints.py

```diff
@@ -45,22 +46,21 @@
 class ModelEndpoints:
     """Provide different methods for handling model endpoints such as listing, writing and deleting"""
 
-    def create_or_patch(
+    def create(
         self,
```

Comment:
Since we are calling other static function from this method, we can't use it as a static function. However, I will change it to a `classmethod` which is more accurate in this case.